_helpers/update-issue-templates.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs').promises;
+const path = require('path');
+
+const OUTPUT_PATH = path.resolve(
+  __dirname,
+  '..',
+  '.github',
+  'ISSUE_TEMPLATE',
+  'template_bug_report.yml'
+);
+const INPUT_PATH = path.resolve(__dirname, 'template_bug_report.yml');
+const TEMPLATES_FILE = path.resolve(__dirname, '..', 'templates.json');
+
+async function getTemplateNames() {
+  const templatesContent = JSON.parse(
+    await fs.readFile(TEMPLATES_FILE, 'utf8')
+  );
+  return templatesContent.templates.map((t) => t.name);
+}
+
+async function run() {
+  const templates = await getTemplateNames();
+
+  const optionsList = templates
```

Comment:
Might be an idea to sort the templates so that it's easier to find the one you want to reference.